projects/components/src/checkbox/checkbox.component.ts

```diff
@@ -13,7 +13,7 @@ import { MatCheckboxChange } from '@angular/material/checkbox';
       [disabled]="this.disabled"
       class="htc-checkbox"
     >
-      <htc-label class="label" *ngIf="this.label" [label]="this.label"></htc-label>
+      <htc-label class="label" *ngIf="this.label !== undefined" [label]="this.label"></htc-label>
```

Comment:
Why switch to only undefined? It can be empty as well.

Reply:
The previous check was not working for 0, so refined it. I can add a check for empty strings as well.